src/sec_edgar_bulker/core/models.py

```
@@ -0,0 +1,499 @@
+"""Pydantic models for SEC EDGAR Bulker."""
+
+from typing import List, Optional, Dict, Any, Literal
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)